### PR TITLE
When user selects scope aws, disable report mode

### PR DESCRIPTION
**Pull request by jimmytsang**
Problem: Currently we don't support msd policies that are `scopeaws: 'true' && enforcementstate: 'report'`
Solution: 

- when user selects scope 'aws' OR 'all', disable the report mode radio button.
- when user deselects scope 'aws' enable the report mode radio button.

**Initial State of add msd policy modal**
image
**User clicks on scope AWS - select**
image
**User clicks on scope AWS - deselect**
image
**User clicks on scope ALL**
image

### CODE REVIEW

File: ui/src/components/microsegmentation/AddSegmentation.js [hunk 1]
```diff
@@ -1343,18 +1357,55 @@ class AddSegmentation extends React.Component {
...
+                                        name={'scopeawsCheckBox' + i}
+                                        id={'scopeawsCheckBox' + i}
+                                        key={'scopeawsCheckBox' + i}
+                                        label='AWS'
+                                        onChange={(e) =>
+                                            this.handleInputChange(e, i)
+                                        }
+                                    />
+                                </CheckBoxSectionDiv>
+                            </SectionDiv>
```
Comment (jimmytsang) [inline ui/src/components/microsegmentation/AddSegmentation.js:1396]: As discussed with @MartinTrojans , the policy enforcement state radio buttons depend on the user's scope input selection therefore they should come after the scope check boxes. 

**Updated Form Order**
image

Comment (jimmytsang) [reply ui/src/components/microsegmentation/AddSegmentation.js:1396]: The only file changed was `AddSegmentation.js`. All other files are changed from fix-lint

File: ui/src/components/denali/InputDropdown.js [hunk 2]
```diff
@@ -295,7 +295,7 @@ class InputDropdown extends React.Component {
                                     selected:
                                         options.selectedItem &&
                                         options.selectedItem.value ===
-                                        item.value,
+                                            item.value,
```
Comment (MartinTrojans) [inline ui/src/components/denali/InputDropdown.js:298]: Why do we change the indent here?

File: ui/src/components/denali/InputDropdown.js [hunk 3]
```diff
@@ -372,7 +372,7 @@ class InputDropdown extends React.Component {
                                             placeholder: this.props.placeholder,
                                             readOnly: Boolean(
                                                 !this.props.asyncSearchFunc &&
-                                                !this.props.filterable
+                                                    !this.props.filterable
```
Comment (MartinTrojans) [inline ui/src/components/denali/InputDropdown.js:375]: Same thing for the indent

**Review by MartinTrojans — APPROVED** [review]

File: ui/src/components/denali/InputDropdown.js (hunk 2 above)
Comment (jimmytsang) [reply ui/src/components/denali/InputDropdown.js:298]: the `npm run fix-lint` script is a bit funky and changes things back and forth

File: ui/src/components/denali/InputDropdown.js (hunk 3 above)
Comment (jimmytsang) [reply ui/src/components/denali/InputDropdown.js:375]: same as above

**Review by MartinTrojans — APPROVED** [review]

**Pull request merged**